### PR TITLE
make sure all files are updates with new version

### PR DESCRIPTION
**Pull request by remibergsma**
During the previous release I had to edit these manually. This will fix that so all versions get updated.

### CODE REVIEW

File: tools/build/build_asf.sh [hunk 1]
```diff
@@ -101,6 +101,11 @@ perl -pi -e "s/-SNAPSHOT//" build/replace.properties
 perl -pi -e "s/-SNAPSHOT//" services/console-proxy/plugin/pom.xml
 perl -pi -e "s/-SNAPSHOT//" tools/marvin/setup.py
 perl -pi -e "s/-SNAPSHOT//" tools/marvin/marvin/deployAndRun.py
+perl -pi -e "s/-SNAPSHOT//" debian/changelog
+perl -pi -e "s/-SNAPSHOT//" services/iam/plugin/pom.xml
```
Comment (DaanHoogland) [inline tools/build/build_asf.sh]: to bad the poms don't get changed by the mvn versions:set task :(

**Comment by DaanHoogland** [comment]
Looks good to me. The version-branch should have gone in like this from the get go.

File: tools/build/build_asf.sh [hunk 2]
```diff
@@ -101,6 +101,11 @@ perl -pi -e "s/-SNAPSHOT//" build/replace.properties
...
+perl -pi -e "s/-SNAPSHOT//" services/iam/pom.xm
+perl -pi -e "s/-SNAPSHOT//" services/iam/server/pom.xml
```
Comment (pdube) [inline tools/build/build_asf.sh]: Why do you have to add these manually? What about the other folders in /services?

File: tools/build/build_asf.sh [hunk 3]
```diff
@@ -123,11 +128,7 @@ git clean -f
 
 #create a RC branch
 RELEASE_BRANCH="RC"`date +%Y%m%dT%H%M`
```
Comment (pdube) [inline tools/build/build_asf.sh]: Might be clearer to name this variable RC_BRANCH_SUFFIX

**Comment by remibergsma** [comment]
The build script now makes a branch that makes sense: `4.6.1-RC20151130T2206` instead of `4.6-RC20151130T2206`. Also the versions are updated properly.

Output example:
screen shot 2015-11-30 at 22 07 02

File: tools/build/build_asf.sh (hunk 2 above)
Comment (remibergsma) [inline tools/build/build_asf.sh]: It is supposed to go automatically, but it doesn't pick these ones up. It should be investigated why that is and if these are still used.

File: tools/build/build_asf.sh (hunk 3 above)
Comment (remibergsma) [inline tools/build/build_asf.sh]: Yep, changed it.

**Comment by pdube** [comment]
Ok then, LGTM

**Comment by rafaelweingartner** [comment]
It LGTM, 
I would just recommend using “sed” in combination with “find” to look up for every file that we want to change the version number, instead of perl, but that is more a matter of taste.

**Comment by remibergsma** [comment]
@rafaelweingartner I agree and I think this needs some TLC indeed. Investigate why it doesn't work with mvn and then do one global search/replace for the remaining. I'll merge this now and proceed with 4.6.1RC, then make the improvement later.

**Comment by pdion891** [comment]
LGTM

**Comment by rafaelweingartner** [comment]
@remibergsma , I think it would have been a good idea to have squashed your commits

**Pull request merged**